src/MagicWormhole/Internal/Rendezvous.hs

```diff
@@ -105,18 +105,23 @@ runClient
   => WebSocketEndpoint -- ^ The websocket to connect to
   -> Messages.AppID -- ^ ID for your application (e.g. example.com/your-application)
   -> Messages.Side -- ^ Identifier for your side
+  -> Maybe Socket.Socket -- ^ Whether to use the given socket for the websocket connection
```

Comment:
The comment here doesn't read easily for me.  "Whether" makes me think it should be a boolean.  Perhaps something like "Just an existing socket to use or Nothing to create and use a new one."